modules/op-deploying-api-service-for-tekton-hub.adoc

```diff
@@ -0,0 +1,186 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="deploying-the-api-service-for-tekton-hub_{context}"]
+= Deploying the API service for {tekton-hub}
+
+[discrete]
+.Prerequisites
+* Ensure that you are in the `<project_root>/config/` directory, where `<project_root>` is the top level directory of the cloned {tekton-hub} repository.
+
+[discrete]
+.Procedure
+. Set up route or ingress for the API.
++
+[source,terminal]
+----
+$ oc apply -f 04-openshift/40-api-route.yaml -f 04-openshift/40-auth-route.yaml -n tekton-hub
+----
+
+. Create an OAuth application with your Git repository hosting service provider.
+* For a GitHub OAuth application, set the `Homepage URL` and the `Authorization callback URL` as `<auth-route>`.
+* For a GitLab OAuth application, set the `REDIRECT_URI` as `<auth-route>/auth/gitlab/callback`.
+* For a BitBucket OAuth application, set the `Callback URL` as `<auth-route>`.
+
+. Edit the `02-api/20-api-secret.yaml` file to update the API secret.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tekton-hub-api
+  namespace: openshift-pipelines <1>
+type: Opaque
+stringData:
+  GH_CLIENT_ID: '' <2>
+  GH_CLIENT_SECRET: '' <3>
+  GHE_URL: '' <4>
+  GL_CLIENT_ID: '' <5>
+  GL_CLIENT_SECRET: '' <6>
+  GLE_URL: '' <7>
+  BB_CLIENT_ID: '' <8>
+  BB_CLIENT_SECRET: '' <9>
+  JWT_SIGNING_KEY: '' <10>
+  ACCESS_JWT_EXPIRES_IN: '' <11>
+  REFRESH_JWT_EXPIRES_IN: '' <12>
+  AUTH_BASE_URL: '' <13>
+----
+<1> The target namespace mentioned in the `TektonHub` CR; default is `openshift-pipelines`.
+<2> The Client ID from the GitHub OAuth application.
+<3> The Client Secret from the GitHub OAuth application.
```

Comment:
```suggestion
<2> The client ID from the GitHub OAuth application.
<3> The client secret from the GitHub OAuth application.
```
Unless you're referencing the UI labels in an application (in which case, should they be bold), these should be lower case. Multiple instances.